test/cmdlineTests/standard_immutable_references_function_pointer/output.json

```diff
@@ -0,0 +1,2 @@
+{"contracts":{"a.sol":{"A":{"evm":{"deployedBytecode":{"immutableReferences":{"a.sol:A.x (10) address":[73],"a.sol:A.x (10) functionIdentifier":[106]},"linkReferences":{},"object":"bytecode removed","opcodes":"opcodes removed","sourceMap":"0:110:0:-:0;;;;5:9:-1;2:2;;;27:1;24;17:12;2:2;0:110:0;;;;;;;;;;;;;;;;;;;;;12:1:-1;9;2:12;80:28:0;;;:::i;:::-;;13:24;;;:::i;:::-;;80:28;102:1;;:3;;;;;;;;;;;;;;;;;;;;;;;5:9:-1;2:2;;;27:1;24;17:12;2:2;102:3:0;;;;8:9:-1;5:2;;;45:16;42:1;39;24:38;77:16;74:1;67:27;5:2;102:3:0;;;;80:28::o;13:24::-;:::o"}}}}},"errors":[{"component":"general","formattedMessage":"a.sol: Warning: Source file does not specify required compiler version!
```

Comment:
Yeah - here the stack slot name is also appended to it...

Reply:
actually, the value `73` will change with changes to the code generator :)

Reply:
Hopefully that won't happen all the time :-).